documentation/images/plantuml/agent/flows/sta_steering.puml

```diff
@@ -0,0 +1,19 @@
+@startuml
+->controller: dev_reset_default
+->controller: Prepare setup( execute 4.1.2 test)
+controller->backhaul_manager: send_cmdu(Backhaul Steering Request)
+backhaul_manager->backhaul_manager: handle_1905_msg(\n Backhaul Steering Request message,\n Backhaul Steering Request TLV(0x9E))
+backhaul_manager->controller: send_cmdu(ACK)
+backhaul_manager->controller: send_cmdu(Associated Request)
+controller->controller: \nhandle_1905_msg(Associated Request)
+controller->backhaul_manager: send_cmdu(Associated Response)
+backhaul_manager->backhaul_manager: handle_1905_msg(Associated Response)
```

Comment:
As mentioned orally:
- it's not a 1905 message, so not `send_cmdu` and `handle_1905_msg`; rather, it should go through bwl;
- it's not to the controller, but to the bAP to which it's connected;
- it's Association, not Associated.

Reply:
@arnout fixed. Updated the branch.